projects/NAB/README.md

```diff
@@ -0,0 +1,11 @@
+# Description
+
+Experimental [NAB](https://github.com/numenta/NAB) detectors.
+
+Requires NAB to be installed.
+
+~~~
+python run.py -d null distalTimestamps1CellPerColumn --dataDir ~/nta/NAB/data --windowsFile ~/nta/NAB/labels/combined_windows.json --profilesFile ~/nta/NAB/config/profiles.json --detect --score --normalize
```

Comment:
Yeah, it works. And it's consistent with the `run.py --help` output `[-d [DETECTORS [DETECTORS ...]]]`